packages/atomic-hosted-page/package.json

```diff
@@ -21,9 +21,8 @@
   ],
   "scripts": {
     "clean": "rimraf -rf dist/*",
-    "build": "npm run build:stencil && npm run validate:definitions",
-    "build:stencil": "node --max_old_space_size=6144 ../../node_modules/@stencil/core/bin/stencil build",
-    "dev": "npm run build:stencil -- --dev --watch --serve",
+    "build": "nx build",
```

Comment:
If, from the root of the repo, you do `nx run atomic-hosted-page:build`, and then, you cd into `packages/atomic-hosted-page` and run `npm run build`, the cache will **not** be re-used.

Explanation: I was able to verify that Nx's hash calculations are impacted by the current CWD.
* Changing the command itself never seems to make a difference.
  * From the Atomic package, `nx build` and `nx run atomic:build` share the same cache, which is different from the cache of `nx run atomic:build` at the root of the repo.
  * The cache is always added to the root `node_modules`, never in sub-packages.
* I didn't find the cause of this in their source code, they don't seem to explicitly cache the CWD path.
* Nx does cache the dependency graph, but when I do `nx graph --file=somefile.json`, changing the CWD doesn't cause any change.
* I was able to "fix" this by creating my own small script which executes a command with a different CWD, but I decided not to include it in this PR. I will revisit this later.

Reply:
ngl: Don't care much about caching on local builds. What matter is the CI/CD, where the CWD is predictable (no hooman, no troubles).
So, in short... we don't care?

Reply:
One flaw I forgot to mention is that running `npm run -w @coveo/somepackage` changes the CWD.

So, if you just did `npm run build` at the repo root, and then you did `npm run build -w @coveo/atomic`, then the cache would miss.

I guess that whether this is important depends on whether some people do this stuff often. Using `nx run` instead of `npm run` solves the issue.

Reply:
Fair, worst case we can change it in the future